src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/AccountAuthorizedDepositorsProcessor.cs

```diff
@@ -75,7 +75,7 @@
 
 namespace RadixDlt.NetworkGateway.PostgresIntegration.LedgerExtension;
 
-internal record struct AccountAuthorizedDepositorsDbLookup(long AccountEntityId, long ResourceEntityId, long? NonFungibleIdDataId);
+internal record struct AccountAuthorizedDepositorsDbLookup(long AccountEntityId, long ResourceEntityId, string? NonFungibleId);
```

Comment:
This and the correlated SQL query might blow into our faces later on. Maybe we should treat both variants separately from day one?